packages/samples/vuejs/package.json

```diff
@@ -17,8 +17,9 @@
     "@vue/cli-plugin-babel": "~5.0.0",
     "@vue/cli-plugin-typescript": "~5.0.0",
     "@vue/cli-service": "~5.0.0",
-    "typescript": "~4.5.5",
+    "cypress": "^9.5.2",
+    "html-loader": "^3.1.0",
     "ncp": "^2.0.0",
-    "html-loader": "^3.1.0"
+    "typescript": "~4.5.5"
```

Comment:
We have typescript in the root package.json devDependencies, we might not need it here I think